core/src/main/scala/org/apache/spark/internal/io/HadoopMapReduceCommitProtocol.scala

```diff
@@ -57,6 +57,15 @@ class HadoopMapReduceCommitProtocol(jobId: String, path: String)
    */
   private def absPathStagingDir: Path = new Path(path, "_temporary-" + jobId)
 
+  /**
+   * Checks whether there are files to be committed to an absolute output location.
+   *
+   * As the committing and aborting the job occurs on driver where `addedAbsPathFiles` is always
+   * null, it is necessary to check whether the output path is specified, that may not be the case
+   * for committers not writing to distributed file systems.
+   */
+  private def hasAbsPathFiles: Boolean = path != null
```

Comment:
Please add `@param path` to line 38 for explaining it.

Reply:
done